hassio/src/backups/hassio-backups.ts

```diff
@@ -127,6 +127,14 @@ export class HassioBackups extends LitElement {
           html`${entry || backup.slug}
             <div class="secondary">${backup.secondary}</div>`,
       },
+      size: {
+        title: this.supervisor?.localize("backup.size") || "",
```

Comment:
You should be sure the supervisor is set here? We should be able to do `this.supervisor!` ?

Reply:
Or have the variable definition be set with !

Reply:
Operator is not needed at all, we don't get here without the supervisor object, adjusted all instances 👍